Runtime/Scripts/SceneExporter/ExporterAnimationPointer.cs

```diff
@@ -95,204 +76,40 @@ public void AddAnimationData(Object animatedObject, string propertyName, GLTFAni
 			string extensionName = null;
 			var propertyType = values[0]?.GetType();
 
+			var animationPointerExportContext = _plugins.FirstOrDefault(x => x is AnimationPointerExportContext) as AnimationPointerExportContext;
+			
 			switch (animatedObject)
 			{
 				case Material material:
 					// Debug.Log("material: " + material + ", propertyName: " + propertyName);
 					// mapping from known Unity property names to glTF property names
-					switch (propertyName)
+					if (animationPointerExportContext == null ||
+					    animationPointerExportContext.materialPropertiesRemapper == null)
 					{
-						case "_Color":
```

Comment:
Are these alternative property names still supported or did that support get lost in this PR?
As in: when animating e.g. an URP/Lit material, will that still export correctly with KHR_animation_pointer?

Reply:
Ah, I see it in DefaultMaterialPropertiesRemapper